spec/support/coronavirus_feature_steps.rb

```diff
@@ -344,16 +350,48 @@ def then_a_new_timeline_entry_is_created
 # Editing timeline entries
 
 def when_i_visit_the_edit_timeline_entry_page
-  visit "/coronavirus/landing/timeline_entries/#{@timeline_entry.id}/edit"
+  visit "/coronavirus/landing/timeline_entries/#{@timeline_entry_one.id}/edit"
 end
 
 def and_i_see_the_existing_timeline_entry_data
-  expect(page).to have_selector("input[value='#{@timeline_entry.heading}']")
-  expect(page).to have_content(@timeline_entry.content)
+  expect(page).to have_selector("input[value='#{@timeline_entry_one.heading}']")
+  expect(page).to have_content(@timeline_entry_one.content)
 end
 
 def then_the_timeline_entry_is_updated
-  expect(@timeline_entry.reload.content).to eq("##Form content")
+  expect(@timeline_entry_one.reload.content).to eq("##Form content")
+end
+
+# Reordering timeline entries
+
+def when_i_visit_the_reorder_timeline_entries_page
+  visit "/coronavirus/landing/timeline_entries/reorder"
+end
+
+def then_i_see_the_timeline_entries_in_order
+  element = find("#step-0").find(".step-by-step-reorder__step-title")
+  expect(element).to have_content @timeline_entry_one.heading
+
+  element = find("#step-1").find(".step-by-step-reorder__step-title")
+  expect(element).to have_content @timeline_entry_two.heading
```

Comment:
I think you could do this check with a single assertion along the lines of:

`expect(page).to have_content(/#{@timeline_entry_one.heading}.*#{@timeline_entry_two.heading}/, normalize_ws: true)` which would check that the headers appear in the particular ordering.